apps/crn-server/src/data-providers/contentful/calendar.data-provider.ts

```diff
@@ -160,6 +160,12 @@ export class CalendarContentfulDataProvider implements CalendarDataProvider {
 
     updateEntryFields(calendar, otherFields);
 
+    if ('googleApiMetadata' in update) {
```

Comment:
why do we need this exactly? looks like we are bailing out of the regular update if there is any metadata to update, but why do you think we should do that? technically someone might want to update all fields if they feel like it, and looking at the interface it seems perfectly fine. We would need to update the typings if we want to say 'you can update google metadata or the remaining fields but not both'.

Reply:
I thought about using this `googleApiMetadata` field only if we want to set it to null
![Screenshot 2023-12-06 at 12 55 53](https://github.com/yldio/asap-hub/assets/16595804/f7bdf97c-6228-41fb-88f6-8703e948bef7)


> looks like we are bailing out of the regular update if there is any metadata to update
I thought that it would be nice to return if `googleApiMetadata` is present because for me it doesn't make sense to update `googleApiMetadata` and `recordId` at the same time since recordId is inside `googleApiMetadata`